cmake/ConkyPlatformChecks.cmake

```diff
@@ -469,16 +469,15 @@ if(BUILD_X11)
     set(luaimlib2_libs ${IMLIB2_LIBS} ${IMLIB2_LDFLAGS} ${LUA_LIBRARIES})
     set(luaimlib2_includes
       ${IMLIB2_INCLUDE_DIRS}
-      ${LUA_INCLUDE_DIR}
-      ${X11_INCLUDE_DIR})
```

Comment:
Not sure why luaimlib2 had X11 in includes, it should be separate. All of these Lua features are now under BUILD_GUI instead of BUILD_X11 as they don't require/depend on X11 at all.